lib/ash/api/diagram/diagram.ex

```diff
@@ -123,6 +123,15 @@ defmodule Ash.Api.Info.Diagram do
     Enum.map_join(list, "\n", fn item -> "#{indent}#{indent}#{template_fn.(item)}" end)
   end
 
+  defp list_arguments([]), do: ""
+
+  defp list_arguments(arguments) do
+    arguments
+    |> Enum.map(&"#{class_short_type(&1.type)} #{&1.name}")
```

Comment:
You'll want to use `Enum.map_join` like above. Credo whinges about this as I discovered.

Reply:
Fixed, thanks